lldb/source/Target/Statistics.cpp

```diff
@@ -307,7 +311,8 @@ llvm::json::Value DebuggerStats::ReportStatistics(Debugger &debugger,
       {"totalModuleCount", num_modules},
       {"totalModuleCountHasDebugInfo", num_modules_has_debug_info},
       {"totalModuleCountWithVariableErrors", num_modules_with_variable_errors},
-      {"totalModuleCountWithIncompleteTypes", num_modules_with_incomplete_types},
+      {"totalModuleCountWithIncompleteTypes",
+       num_modules_with_incomplete_types},
```

Comment:
revert non related whitespace change

Reply:
No, it is actually required by the formatter reported by github: 
https://github.com/llvm/llvm-project/actions/runs/7789051206/job/21239805569?pr=80375